src/formats/sprite.c

```diff
@@ -308,7 +310,12 @@ int sd_sprite_vga_decode(sd_vga_image *dst, const sd_sprite *src) {
             case 1:
                 while(data > 0) {
                     uint8_t b = src->data[i];
-                    int pos = ((y * src->width) + x);
+                    unsigned int pos = ((y * src->width) + x);
+                    if(pos >= dst_size) {
```

Comment:
Can you comment exactly what we're checking for here?

Reply:
Added a comment on dst_size and on the if, explaining that this is bounds checking for the `dst` buffer.